src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main-page">
+      <div class="calendar calendar--start-day--sun calendar--month-length--29">
+        <div class="calendar__day-box calendar__day-box--1"></div>
+        <div class="calendar__day-box calendar__day-box--2"></div>
+        <div class="calendar__day-box calendar__day-box--3"></div>
+        <div class="calendar__day-box calendar__day-box--4"></div>
+        <div class="calendar__day-box calendar__day-box--5"></div>
+        <div class="calendar__day-box calendar__day-box--6"></div>
+        <div class="calendar__day-box calendar__day-box--7"></div>
+        <div class="calendar__day-box calendar__day-box--8"></div>
+        <div class="calendar__day-box calendar__day-box--9"></div>
+        <div class="calendar__day-box calendar__day-box--10"></div>
+        <div class="calendar__day-box calendar__day-box--11"></div>
+        <div class="calendar__day-box calendar__day-box--12"></div>
+        <div class="calendar__day-box calendar__day-box--13"></div>
+        <div class="calendar__day-box calendar__day-box--14"></div>
+        <div class="calendar__day-box calendar__day-box--15"></div>
+        <div class="calendar__day-box calendar__day-box--16"></div>
+        <div class="calendar__day-box calendar__day-box--17"></div>
+        <div class="calendar__day-box calendar__day-box--18"></div>
+        <div class="calendar__day-box calendar__day-box--19"></div>
+        <div class="calendar__day-box calendar__day-box--20"></div>
+        <div class="calendar__day-box calendar__day-box--21"></div>
+        <div class="calendar__day-box calendar__day-box--22"></div>
+        <div class="calendar__day-box calendar__day-box--23"></div>
+        <div class="calendar__day-box calendar__day-box--24"></div>
+        <div class="calendar__day-box calendar__day-box--25"></div>
+        <div class="calendar__day-box calendar__day-box--26"></div>
+        <div class="calendar__day-box calendar__day-box--27"></div>
+        <div class="calendar__day-box calendar__day-box--28"></div>
+        <div class="calendar__day-box calendar__day-box--29"></div>
+        <div class="calendar__day-box calendar__day-box--30"></div>
+        <div class="calendar__day-box calendar__day-box--31"></div>
```

Comment:
Those divs should have only one class name - calendar__day-box, get rid of the other one. You can solve it without it